src/config/scrivitoSites.ts

```diff
@@ -110,6 +110,10 @@ function rootContentId() {
     ?.contentId()
 }
 
+export function portalSiteAvailable(): boolean {
+  return !!rootContentId()
+}
+
```

Comment:
* [ ] Instead of introducing a custom helper we could use `Scrivito.currentSiteId() !== null` for "site available".